test/CMakeLists.txt

```diff
@@ -7,23 +7,9 @@ aux_source_directory(libsolidity SRC_LIST)
 aux_source_directory(contracts SRC_LIST)
 aux_source_directory(liblll SRC_LIST)
 
-get_filename_component(TESTS_DIR "${CMAKE_CURRENT_SOURCE_DIR}" ABSOLUTE)
+list(REMOVE_ITEM SRC_LIST "./fuzzer.cpp")
 
-# search for test names and create ctest tests
```

Comment:
This generated some auxiliary information so that ctest could be used as a test runner. The main advantage of ctest was that tests could be run in parallel. Since this is not possible anymore because of the IPC connection required, I did not see any reason to keep it.